### ISSUE TITLE
Fix and update data on coinmarketcap

### ISSUE THREAD

**Issue by xaur** [xaur/decred-issues#72]
https://coinmarketcap.com/currencies/decred/

CMC still links to Insight (mainnet.decred.org) as a primary explorer, perhaps that's why people keep talking/asking about it. Update CMC to de-prioritize or even remove Insight.

If there are other "market cap" sites that link to Insight, list them here and we'll update this task or create new ones.

**Comment by sambiohazard** [comment]
Will this also go to Ditto, as its correction of information?

**Comment by xaur** [comment]
Good question. I'll ask them. Also pinging @sumiflow who contacted many such sites to correct Decred supply information.

**Comment by sumiflow** [comment]
I agree that dcrdata is the better site, but CMC and others probably do this because mainnet.decred.org looks more official since it is under the decred.org domain.  IMO, before contacting CMC, perhaps we should A. move dcrdata to the decred.org domain, or B. eventually drop the older mainnet site completely.  I vote option B.  Perhaps @chappjc has an opinion since he did most of the work on dcrdata.

**Comment by devwarrior777** [comment]
Re: Ditto .. if they have influence where we (stakeholders) do not then I would respectfully ask them to get involved. Thanks.

**Comment by xaur** [comment]
Ditto is [not pursuing](https://matrix.to/#/!OfChXgczrIlpEZSFAv:decred.org/$15474897963391hhDDU:decred.org) this:

> Since CoinMarketCap isn't considered a media outlet, it's not something we have much recourse to. We'll mostly be focused on media.

If anybody would like to reach CMC for this, please proceed.

**Comment by sambiohazard** [comment]
I was looking to submit the request but saw more outdated/missing info

- They link to forum as a message board, which is now archived.
- Slack and Discord are linked as chat platforms but Matrix is missing.

Should we consider this for update as well? My suggestion

- Replace forum with reddit
- ask for adding Matrix link as third chat option.

**Comment by xaur** [comment]
+1 forum -> reddit

I'd advocate Matrix as primary (yes I'm biased), Slack or Discord as secondary and have https://decred.org/community/ somewhere

**Comment by xaur** [comment]
From [chat](https://matrix.to/#/!MgQoetFiyjrHAywokv:decred.org/$15495707532171MGsdE:decred.org):

* can add link to politeia as Message Board
* docs.decred.org is duplicated as 'Website 2" and "Technical Documentation", decide keep or fix
* find out if to keep Stats 3 (https://stats.decred.org/)
* medium blog is not message board but is an important link, link it as Blog if possible
* update About, as a minimum, Decred is not a Bitcoin fork

**Comment by sambiohazard** [comment]
ok submitted the content of this gist: https://gist.github.com/sambiohazard/82e77e08be460ebc27823b1a7e8ad818

optional email was asked but i don't know if we have a support email so submitted without it.

**Comment by xaur** [comment]
Good idea to place Medium blog as website, I didn't think of that. Also good to have this gist, so if they don't react or reply we can submit it again.

**Comment by sambiohazard** [comment]
They fixed only one thing. They updated Website 2 from docs to politeia. rest of the links and description is still the same.

I think it will be more effective if someone who is known decred contributor like jz or haon can contact with a email id. email will also help if they have questions about source or want to notify.

**Comment by xaur** [comment]
Ok thanks for putting together the request and giving a first try.

@jzbz or @noahpierau can you help us with sending the request in sambiohazard's [gist](https://gist.github.com/sambiohazard/82e77e08be460ebc27823b1a7e8ad818) from a serious looking official business pro suit VIP email?

**Comment by jzbz** [comment]
I have a contact there, I will forward him the content of the gist.

**Comment by xaur** [comment]
[This snippet](https://github.com/xaur/decred-issues/issues/79#issuecomment-466715647) may be also useful for Description field.

**Comment by sambiohazard** [comment]
any update on this? @jzbz

**Comment by xaur** [comment]
CMC still calls Decred "Bitcoin fork".

**Comment by jzbz** [comment]
I know, will get them to change it once we have our final messaging nailed down.

**Comment by xaur** [comment]
Now our [messaging](https://github.com/decredcommunity/pr/blob/release/foundational-messaging.md) is in a good shape.

CMC updated our page to add the Tokenview block explorer (which added DCR in Sep 2019). I guess it was per Tokenview's request and it is annoying that they keep ignoring ours.

CMC's text assets are captured [here](https://github.com/decredcommunity/textassets/tree/master/coinmarketcap) - anybody, feel free to submit a change if you manage to update the data. We really need more hands on keeping Decred presence updated on the growing number of sites (#140).

**Comment by chappjc** [comment]
WTF... tokenview?  Maybe they are reading this issue for laughs.

**Comment by xaur** [comment]
I don't mind Tokenview - apparently it gives us more exposure in China. It's just not cool to process some requests and ignore the rest.
